filament/src/components/TransformManager.h

```diff
@@ -205,11 +205,11 @@ class UTILS_PRIVATE FTransformManager : public TransformManager {
             math::mat4f,    // world
             math::float3,   // accurate local translation
             math::float3,   // accurate world translation
-            math::mat3f,    // local orientation
-            math::mat3f,    // orientation
+            math::quatf,   // local orientation quaternion
+            math::quatf,   // orientation quaternion
```

Comment:
nit: indentation change is not needed

Reply:
https://github.com/shapr3d/filament/pull/162/commits/6b05fb0008899ce758a2cd8cfc8e1e4b2f094c46